src/containers/pages/IRS/JurisdictionsReport/helpers.ts

```diff
@@ -0,0 +1,308 @@
+        Header: '',
+        accessor: 'perctvisareaseffect',
+        className: 'indicator centered',
+      },
+    ],
+  },
+  {
+    Header: 'Total Structures',
+    columns: [
+      {
+        Header: '',
+        accessor: 'totstruct',
+      },
+    ],
+  },
+  {
+    Header: 'Targeted Structures',
+    columns: [
+      {
+        Header: '',
+        accessor: 'targstruct',
+      },
+    ],
+  },
+  {
+    Header: 'Sprayed Structures',
+    columns: [
+      {
+        Header: '',
+        accessor: 'sprayedstruct',
+      },
+    ],
+  },
+  {
+    Header: 'Spray coverage of targeted (Progress)',
+    columns: [
+      {
+        Cell: (cell: CellInfo) => getIRSThresholdAdherenceIndicator(cell),
+        Header: '',
+        accessor: 'spraycovtarg',
+        className: 'indicator centered',
+      },
+    ],
+  },
+  {
+    Header: 'Room coverage of structures sprayed',
+    columns: [
+      {
+        Cell: (cell: CellInfo) => getIRSThresholdAdherenceIndicator(cell),
+        Header: '',
+        accessor: 'roomcov',
+        className: 'indicator centered',
+      },
+    ],
+  },
+];
+
+/** columns for Namibia IRS focus (spray) areas */
+export const ZambiaFocusAreasColumns = [
+  {
+    Header: 'Name',
+    columns: [
+      {
+        Header: '',
+        accessor: 'jurisdiction_name',
+        minWidth: 180,
+      },
+    ],
+  },
+  {
+    Header: 'Structures on the ground',
+    columns: [
+      {
+        Header: '',
+        accessor: 'totstruct',
+      },
+    ],
+  },
+  {
+    Header: 'Found',
+    columns: [
+      {
+        Header: '',
+        accessor: 'foundstruct',
+      },
+    ],
+  },
+  {
+    Header: 'Sprayed',
+    columns: [
+      {
+        Header: '',
+        accessor: 'sprayedstruct',
+      },
+    ],
+  },
+  {
+    Header: 'Spray Coverage (Effectiveness)',
+    columns: [
+      {
+        Cell: (cell: CellInfo) => getIRSThresholdAdherenceIndicator(cell),
+        Header: '',
+        accessor: 'spraycov',
+        className: 'indicator centered',
+      },
+    ],
+  },
+  {
+    Header: 'Found Coverage',
+    columns: [
+      {
+        Cell: (cell: CellInfo) => getIRSThresholdAdherenceIndicator(cell),
+        Header: '',
+        accessor: 'spraytarg',
+        className: 'indicator centered',
+      },
+    ],
+  },
+  {
+    Header: 'Spray Success Rate (PMI SC)',
+    columns: [
+      {
+        Cell: (cell: CellInfo) => getIRSThresholdAdherenceIndicator(cell),
+        Header: '',
+        accessor: 'spraysuccess',
+        className: 'indicator centered',
+      },
+    ],
+  },
+  {
+    Header: 'Room coverage of structures sprayed',
+    columns: [
+      {
+        Cell: (cell: CellInfo) => getIRSThresholdAdherenceIndicator(cell),
+        Header: '',
+        accessor: 'roomcov',
+        className: 'indicator centered',
+      },
+    ],
+  },
+];
+
+export const IRSTableColumns: { [key: string]: Column[] } = {
```

Comment:
I think having the column definitions in the settings module is tricky because the columns are actual code and may have logic in them and since settings.ts should have as little "dynamic code" as possible, I think having something like `IRSTableColumns` which holds all the defined table columns might be useful.  When we have a clearer idea of how to do dynamic settings, this may be generated somehow by a deployment script.  The intention is to have an object that holds the column defs.  Env vars would then be used to pick particular column defs based on the IRSTableColumns object properties.

Reply:
> settings.ts should have as little "dynamic code" as possible, I think having something like `IRSTableColumns`...

This makes a lot of sense, I am wondering if we will eventually want to abstract it one step further to be something like `ReportTableColumns` to cover different intervention types, but that's probably not yet necessary.

Reply:
> This makes a lot of sense, I am wondering if we will eventually want to abstract it one step further to be something like ReportTableColumns to cover different intervention types, but that's probably not yet necessary.

Yes!  I almost did this but then pulled back so that we focus on IRS for now.  But I think we should do this for the existing FI tables ASAP.